### ISSUE TITLE
Unverified label on verified images

### ISSUE THREAD

**Issue by issiecorvi** [fathomnet/community-feedback#146]
The unverified label is showing up on ALL image thumbnails in explore mode, even those that have been verified to species-level. (Related UX question - when should this label be displayed? Should it be when an image is verified at any level, or only all the way down to species level?)
<img width="1350" alt="Screenshot 2024-01-29 at 3 24 02 PM" src="https://github.com/fathomnet/community-feedback/assets/148128075/acacd712-453f-4ea6-842b-81ceb3bf59ab">
<img width="1339" alt="Screenshot 2024-01-29 at 3 24 38 PM" src="https://github.com/fathomnet/community-feedback/assets/148128075/559d45c6-80b4-43d1-af80-e85591e726b7">

**Comment by issiecorvi** [comment]
Calling on my newbie/non-tech user experience card, I have another idea to toss out to improve the UX around this...

The reason I stumbled upon this bug initially was because I was trying to figure out what exactly the "? unverified" tag meant. At a glance, when first encountering the database not only isn't it not immediately clear, it leaves a negative impression; it doesn't build confidence/trust in the data. 

I've struggled with understanding the need for a secondary, independent verification on all annotations given that they are coming from trusted sources (we're trying to save time, right?). I now understand the definition of gold standard data is this secondary verification by an independent expert. However, practically speaking, the bulk of the data in the database will not receive secondary verification. We want to grow the database more than we want to ensure it's 100% gold standard. That said, we control (to an extent) the sources of the data (and thus, trustworthiness) by granting permissions to upload data and clearly indicating the source. The prevalence of the "? unverified" tag suggests that anyone could be uploading anything and it may not be trustworthy. It discredits trusted sources like MBARI and leaves the impression that you may not want to use "? unverified" data in your work. 

Proposal: to flip the script and instead of tagging unverified data, only **tag verified data** (as "verified" or "gold star" or "verified" with a little star icon in place of the ? icon. FWIW I'm imagining this tag in yellow/gold instead of blue)

From a UX perspective, this would: 
- Reduce doubt/distrust in unverified data (which would have no tag) 
- Increase confidence in verified data (positive tag)
- Help positively motivate community verification efforts (the idea of ADDING gold stars/collecting personal contributions to the gold standard/completion of concept verification is much more positively motivating than "hey look at how much unverified data we have we could really use your help making this better")

![Image](https://github.com/user-attachments/assets/8d59516c-556b-4f3a-81a4-2cdac56e440d)

**Comment by kevinsbarnard** [comment]
As part of the Vue 2 -> 3 migration, I implemented a first cut at this. Here's what it's looking like in the UI.

![Image](https://github.com/user-attachments/assets/8aa6f660-2178-4b48-9e67-fb2d7854ec0e)

This currently works by looking up the bounding boxes for each image result, and if all are verified, then the image gets a green "verified" tag in the grid.

**Comment by issiecorvi** [comment]
I like it! Just to be sure I fully understand - your saying if all boxes for an image meaning any/all concepts (not just the search concept) are verified, then it gets the green label? In other words, it wouldn't appear if the box & label for _C. tanneri_ is verified, but other annotations within the image are not?

**Comment by kevinsbarnard** [comment]
Right, and I recognize that's not ideal-- this is currently a limitation of how the search is being performed. The query says "give me all images that have a <concept> (or a descendant given by <taxonomy provider>) in <marine region> ". The results in the grid are images, _not bounding boxes_. So, marking them is a bit tricky, given verification is per box.

> In other words, it wouldn't appear if the box & label for C. tanneri is verified, but other annotations within the image are not?

Correct, given how I currently have it set up.

Options are:
1. change how we mark an image result as verified/unverified
2. rewrite the search for ROIs instead of images

**Comment by issiecorvi** [comment]
Copy that. I don't know that I have the strongest opinion on if we want to leave it vs. pursue one of these options - perhaps we can add it to the discussion in the next meeting?
